docs/quasar.conf.js

```diff
@@ -7,6 +7,7 @@ module.exports = function (ctx) {
     // --> boot files are part of "main.js"
     boot: [
       { path: 'gdpr', server: false },
+      // { path: 'darkMode', server: true },
```

Comment:
darkMode boot entry commented

Reply:
Yeah. Because it works without it, also in SSR mode. Strangely enough....

Scott